apollo/src/utils/jwt.ts

```diff
@@ -16,30 +16,24 @@ export class JWTValidator {
 
     encodeSession(partialSession: PartialSession): string {
         const issued = Date.now();
-        const expires = issued + (15 * 60);
+        const expires = issued + 15 * 60;
```

Comment:
This still resolves to the correct answer. I don't really like that prettier gets rid of these, wonder if there is a rule we can change for that.